src/dotnet/Chat.UI.Blazor/Components/ChatAudioPanel/AudioPermissionModal.razor

```diff
@@ -1,11 +1,16 @@
 @namespace ActualChat.UI.Blazor.Components
 @using Stl.Extensibility
+@using ActualChat.Hosting
 @implements IModalView<AudioPermissionModal.Model>
 @attribute [MatchFor(typeof(Model), typeof(IModalView))]
 
+@{
+    var canConfigure = HostInfo.AppKind == AppKind.MauiApp;
+}
+
 <DialogFrame Class="service-modal">
     <Body>
-    @if (ScreenSize.IsWide()) {
+    @if (!canConfigure) {
```

Comment:
what will be displayed in case of mobile or browser?

Reply:
In case it's not maui app, the text below will be displayed.